reports/AC187/report_for_AC187_cleanup.rb

```diff
@@ -0,0 +1,60 @@
+require 'archivesspace/client'
+require 'json'
+require 'csv'
+require_relative '../../helper_methods.rb'
+
+aspace_staging_login()
+
+start_time = "Process started: #{Time.now}"
+puts start_time
+
+eadid = "AC187"
+resource_id = "4185"
+output_file = "#{eadid}.csv"
+ao_tree = @client.get("/repositories/4/resources/#{resource_id}/ordered_records").parsed
+#/repositories/:repo_id/resources/:id/ordered_records
+#get_all_archival_objects_for_resource(4, 4185, ['top_containers'])
+
+  CSV.open(output_file, "a",
+    :write_headers=> true,
+    :headers => ["uri", "ref_id", "title", "first_date", "container"]) do |row|
```

Comment:
This is just a nit pick, but row is an odd name for the file accessor. I would have expected something more along the lines of file or output

Reply:
Hmmm. "row" seemed intuitive to me because it iterates through the CSV row by row and does something for each?